CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 ### Fixed
 * <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
 * Fixed queries of min/max/sum/avg on list of primitive mixed. ([#4472](https://github.com/realm/realm-core/pull/4472), never before working)
+* Fixed sort and distinct on dictionary keys, and distinct on dictionary values.
+* Fixed getting a dictionary value which is null.
```

Comment:
What exactly was fixed here - we do have some tests for dictionaries that read a null value and those pass just fine, so would like to extend them to test for whatever this PR fixes and maybe think of other scenarios that I've forgotten.

Reply:
@nirinchev it was `object_store::Dictionary::get<T>(key)` that would crash if the value was null. I will update the changelog to clarify.